internal/api/graphql/graph/model/models.go

```diff
@@ -145,9 +145,24 @@ func NewSeverity(sev entity.Severity) *Severity {
 }
 
 func NewSeverityEntity(severity *SeverityInput) entity.Severity {
-	if severity == nil || severity.Vector == nil {
+	if severity == nil || (severity.Rating == nil && severity.Vector == nil) {
+		// no severity information was passed
 		return entity.Severity{}
 	}
+	if severity.Rating != nil && severity.Vector != nil {
```

Comment:
If a vector is there, let's take the vector by default and simply overrule the "normal" severity. 

@MR2011  @dorneanu  WDYT?

Reply:
I agree

Reply:
I've updated it now so that it just uses the vector and ignores the rating when both are passed.